### PR TITLE
LPS-168563 We should always define rememberMe variable

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi guys,

Can you review this pull request, please?

It is one of those cases when it is easier to see the mistake in the code than trying to reproduce it since, in general, that `rememberMe` variable is not checked because `Liferay.Session` is already defined in the browser, but in case that module has not been loaded yet for whatever reason, `rememberMe` value is checked and in case `ddmFormDisplayContext.isRememberMe()` is returning `false`, the variable is not defined causing a Javascript error and stopping the form to be loaded.

Thanks.

Regards.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp [hunk 1]
```diff
@@ -419,9 +419,7 @@ boolean limitToOneSubmissionPerUser = DDMFormInstanceSubmissionLimitStatusUtil.i
 						</c:choose>
 					}
 
-					<c:if test="<%= ddmFormDisplayContext.isRememberMe() %>">
-						var rememberMe = true;
-					</c:if>
+					var rememberMe = '<%= ddmFormDisplayContext.isRememberMe() %>';
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp:422]: @antonio-ortega can we remove the single quotes here?

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp (hunk 1 above)
Comment (antonio-ortega) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp]: Done!

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:reevaluate:1327401_6940

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**